src/pages/index.astro

```diff
@@ -183,144 +173,36 @@ const metadata = {
     isAfterContent
     items={[
       {
-        title: 'Enhanced user engagement',
+        title: 'Series A Funding Goals',
         description:
-          'Captivate your audience with interactive elements, intuitive navigation, and visually appealing layouts, encouraging longer visits.',
+          'Seeking $25 million in Series A funding to fuel platform development, team expansion, and growth acceleration.',
       },
       {
-        title: 'Continuous improvement',
+        title: 'Industry Growth Trajectory',
         description:
-          'Ensure your website stays aligned with the latest trends and technologies through regular updates and enhancements.',
+          "Rapid industry expansion and high valuations, exemplified by Cognition Labs' $2 billion valuation in just six months, indicate significant market potential.",
       },
       {
-        title: 'Time and resource efficiency',
+        title: 'Future Valuation Potential',
         description:
-          'Skip the time-consuming process of building a website from scratch and launch your online presence sooner with AstroWind.',
-      },
-      {
-        title: 'Community support',
-        description: `Join the growing AstroWind community for insights, resources, and assistance, ensuring you're never alone on your web development journey.`,
+          "Confident in our platform's potential, we aim to achieve a similar or higher valuation within the next 12-18 months.",
       },
     ]}
     image={{
-      src: 'https://images.unsplash.com/photo-1611462985358-60d3498e0364?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80',
+      src: 'https://i.ibb.co/k89md15/10f664a176f07d4a2601d540781321ba.png',
```

Comment:
Raw linking to other's resources such as the png here is a no-no and frowned on. Essentially they pay for hosting and it's prone to breakage. Download all content into our assets folder and load all content from our own servers.

Future todo: Replace this with appropriate image (check with designer) and add to the asset folder.